src/Helpers/ApartmentHelpers.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace BuildEmpire\Apartment\Helpers;
+
+use BuildEmpire\Apartment\Exceptions\SchemaNameNotValidException;
+
+class ApartmentHelpers
+{
+    /**
+     * Does the schema only contain letters and/or numbers.
+     *
+     * @param $schemaName
+     * @return int
+     */
+    public static function isSchemaNameValid($schemaName)
+    {
+        return (preg_match('/^[a-z][a-z0-9_]*$/', $schemaName));
+    }
+
+    /**
+     * Gets the schema and table back in the correct format, and will throw an exception if the schema name is not valid.
+     *
+     * @param $schemaName
+     * @param string $table
+     * @return string
+     * @throws SchemaNameNotValidException
+     */
+    public static function getSchemaTableFormat($schemaName, $table = '')
```

Comment:
Is there a use-case for the empty-string default for `$table` here?  Seems kind of weird and not useful to have as a default.